doc/whats-new.rst

```diff
@@ -38,6 +38,13 @@ Documentation
 Enhancements
 ~~~~~~~~~~~~
 
+- Support for writing netCDF files from xarray datastores (scipy and netcdf4 only)
```

Comment:
Should this be "*to* xarray datastores"?

Reply:
Nevermind, I think it makes sense as is.

Maybe "Support for writing xarray datasets to netCDF files..."

Reply:
It's nice to see you were able to get this to work with SciPy!